src/App.ts

```diff
@@ -260,34 +260,32 @@ export default class App {
       }
     }
 
-    let usingBuiltinOauth = false;
-    if (
-      clientId !== undefined
-      && clientSecret !== undefined
-      && (stateSecret !== undefined || (installerOptions !== undefined && installerOptions.stateStore !== undefined))
-      && this.receiver instanceof ExpressReceiver
-    ) {
-      usingBuiltinOauth = true;
+    let usingOauth = false;
```

Comment:
i think this is a great idea! it might be helpful at some later point to document what the requirements are for a custom receiver that intends to offer installation support. right now, i think the expectation is "the receiver must have an `InstallationProvider` instance at the property named `.installer`", which is good. but it might be worth generalizing even more.